CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 # Change Log
 All notable changes to the "azure-appservice" extension will be documented in this file.
 
+## 0.11.0 - 2018-01-19
+### Added
+🎉 Happy New Year, everybody!  Welcome to 2019! 🎉
+- App settings are hidden by default and can be revealed by clicking on it in the Explorer pane
+- Download remote app settings as `.env` file
+- Upload local `.env` to remote web apps's settings
```

Comment:
"remote app settings" and "remote web apps's settings" isn't consistent. I prefer the first for both